src/functions/utils.php

```diff
@@ -638,4 +638,58 @@ function tribe_register_rest_route( $namespace, $route, $args = array(), $overri
 		$args = apply_filters( 'tribe_register_rest_route_args', $args, $namespace, $route, $override );
 		return register_rest_route( $namespace, $route, $args, $override );
 	}
-}
\ No newline at end of file
+}
+
+if ( ! function_exists( 'tribe_installed_before' ) ) {
+	/**
+	 * Checks if a plugin was installed prior to the passed version.
+	 *
+	 * @since TBD
+	 *
+	 * @param object $class the main plugin class.
+	 * @param string $version a SemVer version string.
+	 * @return boolean
+	 */
+	function tribe_installed_before( $class, $version ) {
+		$pervious_versions = Tribe__Settings_Manager::get_option( $class::VERSION_HISTORY_SLUG );
+		$install_version = !  empty( $pervious_versions[1] ) ? $pervious_versions[1] : $class::VERSION;
```

Comment:
We should create method for this piece that is being repeated; like `tribe_plugin_version_compare()`